screens/Home/MyVcs/IdInputModal.tsx

```diff
@@ -49,104 +49,103 @@ export const IdInputModal: React.FC<IdInputModalProps> = props => {
       headerTitle={t('header')}
       headerElevation={2}>
       <KeyboardAvoidingView
-        style={{flex: 1}}
+        style={Theme.Styles.keyboardAvoidStyle}
         behavior={isIOS() ? 'padding' : 'height'}>
-        <Column fill align="space-between" pY={32} pX={24}>
```

Comment:
Can you please fix styling as I am unable to identify the difference here?

Reply:
same as above